frontend/src/containers/pages/AuthoringTool/CanvasSideBar/ComponentProperties/ButtonPropertiesComponent.jsx

```diff
@@ -71,7 +71,7 @@ export default function ButtonPropertiesComponent({
         </Select>
       </FormControl>
       <FormControl fullWidth className={styles.componentProperty}>
-        <CustomInputLabel shrink>Next Scene</CustomInputLabel>
+        <CustomInputLabel shrink>Linked Scene</CustomInputLabel>
```

Comment:
I'm not sure about this one - might be missing something though. What was the reasoning behind the change?

Reply:
RESOLVED: The button could take us to the previous scene/any scene not necessarily the next scene in the sequence.